backend/tournesol/tests/test_api_rate_later.py

```diff
@@ -138,7 +138,7 @@ def test_authenticated_can_create_with_new_video_id(self):
         client = APIClient()
 
         user = User.objects.get(username=self._user)
-        data = {"video": {"video_id": "newvideo001"}}
+        data = {"entity": {"uid": "yt:newvideo001"}}
```

Comment:
Hello I didn't think of any upgrade strategy. I follow @GresilleSiffle suggestion for upgrading this component

Reply:
Hello @amatissart @farnots 

My fault, I completely forgot about the extension... 

I'm thinking about maintaining both the legacy endpoint with its serializer and the new endpoint with the new serializer, so that both routes cohexist and the extension will continue to work after the front end update.

We will be then able to update the extension. Once deployed on Chrome Web Store and Firefox Add-ons, we can delete the old endpoint.

Reply:
Hello @farnots 

Sorry for the delay, I plan to continue this PR : ) except if you want to take care of it yourself?

Reply:
Ok great thanks @GresilleSiffle , if you need help let me now